modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -42,10 +42,25 @@ this topic and save it as `04_bootstrap.json` in your cluster's installation dir
 describes the bootstrap machine that your cluster requires.
 
 . Export the following variables required by the bootstrap machine deployment:
+
+.. Export the bootstrap URL expiration configuration:
```

Comment:
These don't all need to be separate sections. The `BOOTSTRAP_URL_EXPIRY` and `BOOTSTRAP_URL` variables do not even need to be exported. Can we combine the setting of those two variables in with the "Export the bootstrap ignition variable" section?

Reply:
Yeah, I will combine them. The intention was to call them out individually instead of throwing three different variable sets at once. But I see the whole action is to export the bootstrap ignition, so I'll assume that's the only relevant goal involved here. I will remove the two export prefixes for bootstrap expiry/url if they're not necessary :+1: